docs/index.md

```diff
@@ -6,11 +6,11 @@ An Integreatly Operator can be installed using three different flavours: `manage
 
 To switch between the three you can use export the `INSTALLATION_TYPE` env or use it in conjunction with any of the make commands referenced in this README
 
-### Installed products
+## Installed products
 
 The operator installs the following products:
 
-### managed
+## managed
```

Comment:
The heads managed, managed-api and multitenant-managed-api make more sense to be nested under Installed products. This to have being at fault in the original document.
```suggestion
### managed
```

Reply:
Good point, ty